addons/explosives/functions/fnc_defuseExplosive.sqf

```diff
@@ -17,7 +17,9 @@
 #include "script_component.hpp"
 EXPLODE_2_PVT(_this,_unit,_explosive);
 
-if (getNumber (ConfigFile >> "CfgAmmo" >> typeof _explosive >> "ACE_explodeOnDefuse") == 1) exitWith {
+_explode = random 1.0;
+if (getNumber (ConfigFile >> "CfgAmmo" >> typeOf _explosive >> "ACE_explodeOnDefuse") == 1 &&
+	_explode < GVAR(ExplodeOnDefuseChance)) exitWith {
```

Comment:
Perhaps even allowing the chance of explode on defuse to be defined as part of the explosive. But then we might need to get a bit more indepth with the defusal process to allow ID'ing the explosive and the mode it was set to be triggered in. Allowing a better decision regarding how to go above defusing said explosive.